src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -1,19 +1,42 @@
+import json
+import pathlib
+from datetime import datetime
+
 import src.server.cea_608_encoder.caption_string_utility as utils
 import src.server.cea_608_encoder.scene_utility as scene_utils
+import src.server.config as config
 
 # At a future time this could live in it's own config file
 # Leaving it here temporarily
 supported_caption_formats = [
     'CEA-608'
 ]
 
+def write_caption_data_to_file(caption_data: dict):
+    """Writes caption data to file as json.
 
-def consume(caption_data: dict) -> dict:
+       :param caption_data: json with byte pairs
+    """
+    # datetime object containing current date and time
+    now = datetime.now()
+    # mm.dd.YY_H:M:S
+    dt_string = now.strftime("%m.%d.%Y_%H:%M:%S")
+
+    path = config.path_to_perception
+    path = str(path) + '/data/byte_pairs/'
```

Comment:
can the hard-coded sub-path here become a config too, so if someone wants to change where we write files to they can just change the config variables

Reply:
Done